server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -47,16 +55,25 @@ export class TenantManager implements core.ITenantManager {
             Axios.get<core.ITenantConfig>(`${this.endpoint}/api/tenants/${tenantId}`),
             this.getKey(tenantId)]);
 
-        const getCredentials = async (): Promise<ICredentials> => ({
+        const credentials: ICredentials = {
             password: generateToken(tenantId, null, key, null),
             user: tenantId,
-        });
+        };
+        const defaultQueryString = {
+            token: fromUtf8ToBase64(`${credentials.user}`),
+        };
+        const defaultHeaders = {
+            "Authorization": getAuthorizationTokenFromCredentials(credentials),
+            // TODO: all requests from this GitManager will have the same correlation id. Is this intended behavior?
```

Comment:
I don't think so. Double check with @pradeepvairamani

Reply:
Checked with Pradeep and @PingZhu2232. Ping will follow up with a bug fix PR soon